.gitignore

```diff
@@ -1,5 +1,5 @@
 $ cat .gitignore
 # exclude
-*.jar
```

Comment:
I did – but I had to add a couple of jar files for the compaction hence the more forgiving filter :)

Regards,

Zoran Nikolovski
Lead Software Engineer
e: znikolovski@odecee.com.au<mailto:firstname.lastname@odecee.com.au>
m: +61 4 0801 1215
w: https://au.linkedin.com/in/zorannikolovski

From: David Benge <notifications@github.com<mailto:notifications@github.com>>
Reply-To: AdobeAtAdobe/aem_6-1_docker <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Tuesday, 8 December 2015 at 5:40 PM
To: AdobeAtAdobe/aem_6-1_docker <aem_6-1_docker@noreply.github.com<mailto:aem_6-1_docker@noreply.github.com>>
Cc: Zoran Nikolovski <znikolovski@odecee.com.au<mailto:znikolovski@odecee.com.au>>
Subject: Re: [aem_6-1_docker] Add replication agent to publish (#6)


In .gitignore<https://github.com/AdobeAtAdobe/aem_6-1_docker/pull/6#discussion_r46919534>:

> @@ -1,5 +1,5 @@
>  $ cat .gitignore
>  # exclude
> -*.jar


you did not like my wildcard? ok

—
Reply to this email directly or view it on GitHub<https://github.com/AdobeAtAdobe/aem_6-1_docker/pull/6/files#r46919534>.